x-pack/test/plugin_api_integration/test_suites/event_log/service_api_integration.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function() {
+  describe('Event Log service API', () => {
+    it('should allow logging an event', async () => {});
+  });
+}
```

Comment:
This is here as reference for @YulNaumenko it'll be removed once she picks this up for her PR.